source/common/config/delta_subscription_state.cc

```diff
@@ -11,40 +11,93 @@
 namespace Envoy {
 namespace Config {
 
+namespace {
+
+// Usable with maps and sets. Used for ASSERTs, to avoid too much typing.
+template <typename Container, typename Key>
+bool containerContains(const Container& container, const Key& key) {
```

Comment:
It's a [part of C++20](https://en.cppreference.com/w/cpp/container/unordered_map/contains) and [current master of abseil also seems to implement it](https://github.com/abseil/abseil-cpp/blob/master/absl/container/flat_hash_map.h#L445-L450), but I'm not sure what checkout of abseil we use.

Reply:
I replaced that with `container.contains(key)` - apparently before I was using some older version of abseil that didn't have it, because I was getting build errors.